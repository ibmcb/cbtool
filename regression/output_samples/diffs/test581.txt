|arrival                                    |1374527558                           
|command_originated                         |1374527558                           
|mgt_001_provisioning_request_originated    |1374527558                           
|mgt_002_provisioning_request_sent          |0                                    
|mgt_003_provisioning_request_completed     |0                                    
|uuid                                       |58506942-0CD8-5014-910F-5640CB874DEB 
|ai_33     |nullworkload   |1_x_tinyvm                                        |169.41.211.179  |1374527561     |aidrs_2                                 |D62E8B9F-C83D-50E7-BE25-2C84354F31AA
|ai_34     |nullworkload   |1_x_tinyvm                                        |208.172.172.58  |1374527570     |aidrs_2                                 |8F239CAD-53FB-5E02-A704-CC95EF6A6C3E
|ai_35     |nullworkload   |1_x_tinyvm                                        |93.236.114.211  |1374527576     |aidrs_2                                 |A590BB1E-0A4E-58B7-AD03-C57A3A2DF290
|ai_36     |nullworkload   |1_x_tinyvm                                        |41.53.125.77    |1374527582     |aidrs_2                                 |0ECD9E18-0CE2-5DC0-A28D-DB1D1BB1755F
|ai_37     |nullworkload   |1_x_tinyvm                                        |47.43.132.91    |1374527588     |aidrs_2                                 |D0461107-3AB3-55C0-B98F-DBC788FE0FA9
|vm_122        |tinyvm              |pico32      |169.41.211.179  |simhostb2                  |SUT                 |ai_33   |aidrs_2    |0066F0CD-1B80-53AD-9A99-6601368D1F8C
|vm_123        |tinyvm              |pico32      |208.172.172.58  |simhostb3                  |SUT                 |ai_34   |aidrs_2    |8D7DA588-01C2-57E9-9E33-E96F7781D258
|vm_124        |tinyvm              |pico32      |93.236.114.211  |simhostc3                  |SUT                 |ai_35   |aidrs_2    |00304FCB-DA7D-5CF5-9BBD-11C426610EB8
|vm_125        |tinyvm              |pico32      |41.53.125.77    |simhostc4                  |SUT                 |ai_36   |aidrs_2    |A40D5A9B-1885-58CF-9632-81A6C8C8E75B
|vm_126        |tinyvm              |pico32      |47.43.132.91    |simhosta4                  |SUT                 |ai_37   |aidrs_2    |A31AE3BC-98E8-5C2A-B620-0FE495650FEA
|Used Memory                                          |44.59M                        
|Uptime (in seconds)                                  |13566403                      
|Total Connections Received                           |6495150                       
|Total Commands Processed                             |94144720                      
|Number of Keys                                       |740                           
|Storage Size                                         |16114180080.0                 
|Data Size                                            |13823614784.0                 
|Index Size                                           |550481904                     
|Average Object Size                                  |6600.2460761                  
|Collections                                          |60                            
|D30F1B96-9225-51C9-95FF-3355AD659B11 (simzone_d) VM RESERVATIONS|0                             
|3C091D65-A8D9-5ECF-9C95-36A39D5CAA39 (simzone_a) VM RESERVATIONS|1                             
|671172C3-2E80-5782-AB0C-C348A586106A (simzone_c) VM RESERVATIONS|2                             
|96342978-982B-5D60-9FF3-0C0B3F9C41A9 (simzone_b) VM RESERVATIONS|2                             
|58506942-0CD8-5014-910F-5640CB874DEB (aidrs_2) AI RESERVATIONS|5                             
