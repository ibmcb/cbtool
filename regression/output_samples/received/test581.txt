Going to unconditionally wait for 1m (60 seconds). The command line interface will be blocked during the waiting.
Waited 10 seconds... (16.6666666667%)
Waited 20 seconds... (33.3333333333%)
Waited 30 seconds... (50.0%)
Waited 40 seconds... (66.6666666667%)
Waited 50 seconds... (83.3333333333%)
Waited 60 seconds... (100.0%)
Waited for 60 seconds.
The AIDRS object aidrs_2, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (AIDRS object key)               |value                                
|ai_arrived                                 |30                                   
|ai_arriving                                |0                                    
|ai_departed                                |30                                   
|ai_failed                                  |2                                    
|ai_reservations                            |0                                    
|arrival                                    |1374527558                           
|base_dir                                   |/home/msilva/cloudbench/lib/auxiliary//../..
|cloud_hostname                             |undefined                            
|cloud_ip                                   |undefined                            
|cloud_name                                 |TESTCLOUD                            
|command                                    |aidrsattach TESTCLOUD simplenw empty=empty
|command_originated                         |1374527558                           
|counter                                    |391                                  
|daemon_parallelism                         |50                                   
|detach_parallelism                         |20                                   
|experiment_id                              |REGRESSION_B                         
|iait                                       |6                                    
|identity                                   |/home/msilva/cloudbench/lib/auxiliary//../../credentials/bogus_id_rsa
|lifetime                                   |uniformIXIXI200I300                  
|load_duration                              |uniformIXIXI40I60                    
|load_level                                 |uniformIXIXI1I3                      
|login                                      |klabuser                             
|max_ais                                    |5                                    
|mgt_001_provisioning_request_originated    |1374527558                           
|mgt_002_provisioning_request_sent          |0                                    
|mgt_003_provisioning_request_completed     |0                                    
|model                                      |sim                                  
|name                                       |aidrs_2                              
|notification                               |False                                
|nr_ais                                     |5                                    
|pattern                                    |simplenw                             
|ssh_key_name                               |bogus_id_rsa                         
|state                                      |attached                             
|submitter                                  |True                                 
|temp_attr_list                             |empty=empty                          
|tracking                                   |none                                 
|type                                       |nullworkload                         
|update_attempts                            |720                                  
|update_frequency                           |1                                    
|username                                   |msilva                               
|uuid                                       |58506942-0CD8-5014-910F-5640CB874DEB 
|vm_arrived                                 |117                                  
|vm_arriving                                |0                                    
|vm_departed                                |117                                  
|vm_failed                                  |4                                    
|vm_reservations                            |0                                    
|vmc_arrived                                |16                                   
|vmc_departed                               |12                                   
|vmc_failed                                 |0                                    

The following AIDRSs are attached to this experiment (Cloud TESTCLOUD) :
|name                |pattern                |type              
|aidrs_2             |simplenw               |nullworkload      

The following AIs are attached to this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_33     |nullworkload   |1_x_tinyvm                                        |169.41.211.179  |1374527561     |aidrs_2                                 |D62E8B9F-C83D-50E7-BE25-2C84354F31AA
|ai_34     |nullworkload   |1_x_tinyvm                                        |208.172.172.58  |1374527570     |aidrs_2                                 |8F239CAD-53FB-5E02-A704-CC95EF6A6C3E
|ai_35     |nullworkload   |1_x_tinyvm                                        |93.236.114.211  |1374527576     |aidrs_2                                 |A590BB1E-0A4E-58B7-AD03-C57A3A2DF290
|ai_36     |nullworkload   |1_x_tinyvm                                        |41.53.125.77    |1374527582     |aidrs_2                                 |0ECD9E18-0CE2-5DC0-A28D-DB1D1BB1755F
|ai_37     |nullworkload   |1_x_tinyvm                                        |47.43.132.91    |1374527588     |aidrs_2                                 |D0461107-3AB3-55C0-B98F-DBC788FE0FA9

The following VMs are attached to this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_122        |tinyvm              |pico32      |169.41.211.179  |simhostb2                  |SUT                 |ai_33   |aidrs_2    |0066F0CD-1B80-53AD-9A99-6601368D1F8C
|vm_123        |tinyvm              |pico32      |208.172.172.58  |simhostb3                  |SUT                 |ai_34   |aidrs_2    |8D7DA588-01C2-57E9-9E33-E96F7781D258
|vm_124        |tinyvm              |pico32      |93.236.114.211  |simhostc3                  |SUT                 |ai_35   |aidrs_2    |00304FCB-DA7D-5CF5-9BBD-11C426610EB8
|vm_125        |tinyvm              |pico32      |41.53.125.77    |simhostc4                  |SUT                 |ai_36   |aidrs_2    |A40D5A9B-1885-58CF-9632-81A6C8C8E75B
|vm_126        |tinyvm              |pico32      |47.43.132.91    |simhosta4                  |SUT                 |ai_37   |aidrs_2    |A31AE3BC-98E8-5C2A-B620-0FE495650FEA

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |44.59M                        
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |13566403                      
|Total Connections Received                           |6495150                       
|Total Commands Processed                             |94144720                      
|Number of Keys                                       |740                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.0.6                         
|Storage Size                                         |16114180080.0                 
|Data Size                                            |13823614784.0                 
|Index Size                                           |550481904                     
|Average Object Size                                  |6600.2460761                  
|Collections                                          |60                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |5                             
|SVMs                                                 |0                             
|AIs                                                  |5                             
|AIDRSs                                               |1                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |5                             
|VMs ARRIVED                                          |122                           
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |117                           
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |4                             
|SVM RESERVATIONS                                     |0                             
|SVMs ARRIVED                                         |0                             
|SVMs ARRIVING                                        |0                             
|SVMs DEPARTED                                        |0                             
|SVMs DEPARTING                                       |0                             
|SVMs FAILED                                          |0                             
|AI RESERVATIONS                                      |5                             
|AIs ARRIVED                                          |35                            
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |30                            
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |2                             
|AIDRS RESERVATIONS                                   |1                             
|AIDRSs ARRIVED                                       |2                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |1                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |401                           

 ---------------- VMC-WIDE COUNTERS ----------------
|D30F1B96-9225-51C9-95FF-3355AD659B11 (simzone_d) VM RESERVATIONS|0                             
|3C091D65-A8D9-5ECF-9C95-36A39D5CAA39 (simzone_a) VM RESERVATIONS|1                             
|671172C3-2E80-5782-AB0C-C348A586106A (simzone_c) VM RESERVATIONS|2                             
|96342978-982B-5D60-9FF3-0C0B3F9C41A9 (simzone_b) VM RESERVATIONS|2                             

 ---------------- AIDRS-WIDE COUNTERS ----------------
|58506942-0CD8-5014-910F-5640CB874DEB (aidrs_2) AI RESERVATIONS|5                             

