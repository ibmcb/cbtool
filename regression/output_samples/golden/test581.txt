Going to unconditionally wait for 1m (60 seconds). The command line interface will be blocked during the waiting.
Waited 10 seconds... (16.6666666667%)
Waited 20 seconds... (33.3333333333%)
Waited 30 seconds... (50.0%)
Waited 40 seconds... (66.6666666667%)
Waited 50 seconds... (83.3333333333%)
Waited 60 seconds... (100.0%)
Waited for 60 seconds.
The AIDRS object aidrs_2, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (AIDRS object key)               |value                                
|ai_arrived                                 |30                                   
|ai_arriving                                |0                                    
|ai_departed                                |30                                   
|ai_failed                                  |2                                    
|ai_reservations                            |0                                    
|arrival                                    |1372702716                           
|base_dir                                   |/home/msilva/cloudbench/lib/auxiliary//../..
|cloud_hostname                             |undefined                            
|cloud_ip                                   |undefined                            
|cloud_name                                 |TESTCLOUD                            
|command                                    |aidrsattach TESTCLOUD simplenw empty=empty
|command_originated                         |1372702716                           
|counter                                    |391                                  
|daemon_parallelism                         |50                                   
|detach_parallelism                         |20                                   
|experiment_id                              |REGRESSION_B                         
|iait                                       |6                                    
|identity                                   |/home/msilva/cloudbench/lib/auxiliary//../../credentials/bogus_id_rsa
|lifetime                                   |uniformIXIXI200I300                  
|load_duration                              |uniformIXIXI40I60                    
|load_level                                 |uniformIXIXI1I3                      
|login                                      |klabuser                             
|max_ais                                    |5                                    
|mgt_001_provisioning_request_originated    |1372702716                           
|model                                      |sim                                  
|name                                       |aidrs_2                              
|notification                               |False                                
|nr_ais                                     |5                                    
|pattern                                    |simplenw                             
|ssh_key_name                               |bogus_id_rsa                         
|state                                      |attached                             
|submitter                                  |True                                 
|temp_attr_list                             |empty=empty                          
|tracking                                   |none                                 
|type                                       |nullworkload                         
|update_attempts                            |720                                  
|update_frequency                           |1                                    
|username                                   |msilva                               
|uuid                                       |86CFEE5E-2911-588B-BD22-2741F64C5DCA 
|vm_arrived                                 |117                                  
|vm_arriving                                |0                                    
|vm_departed                                |117                                  
|vm_failed                                  |4                                    
|vm_reservations                            |0                                    
|vmc_arrived                                |16                                   
|vmc_departed                               |12                                   
|vmc_failed                                 |0                                    

The following AIDRSs are attached to this experiment (Cloud TESTCLOUD) :
|name                |pattern                |type              
|aidrs_2             |simplenw               |nullworkload      

The following AIs are attached to this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_33     |nullworkload   |1_x_tinyvm                                        |104.185.137.150 |1372702719     |aidrs_2                                 |B15A7CF1-AE8A-5AC7-844A-9444624D8534
|ai_34     |nullworkload   |1_x_tinyvm                                        |143.39.221.13   |1372702728     |aidrs_2                                 |FD09A200-5473-5B12-A8E1-978BFC73EB25
|ai_35     |nullworkload   |1_x_tinyvm                                        |19.98.52.22     |1372702734     |aidrs_2                                 |B2DA1315-184B-5969-82BE-EACE13E06856
|ai_36     |nullworkload   |1_x_tinyvm                                        |4.23.64.23      |1372702740     |aidrs_2                                 |BDD654ED-1DCF-5C4A-8AC1-C976E8B49794
|ai_37     |nullworkload   |1_x_tinyvm                                        |11.29.62.238    |1372702746     |aidrs_2                                 |19F07BAF-4AF2-5BE0-A03A-19293A169D04

The following VMs are attached to this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_122        |tinyvm              |pico32      |104.185.137.150 |simhostb4                  |SUT                 |ai_33   |aidrs_2    |23B826E1-C849-5E1C-9FB7-5A567BCBB9E7
|vm_123        |tinyvm              |pico32      |143.39.221.13   |simhosta0                  |SUT                 |ai_34   |aidrs_2    |C003215A-B81F-5F5C-A0F5-114230676173
|vm_124        |tinyvm              |pico32      |19.98.52.22     |simhosta1                  |SUT                 |ai_35   |aidrs_2    |589BF7AE-E890-5DF6-99D6-EB593B835162
|vm_125        |tinyvm              |pico32      |4.23.64.23      |simhostb4                  |SUT                 |ai_36   |aidrs_2    |25F2A68D-96CF-5517-808A-BDCB8193F9EA
|vm_126        |tinyvm              |pico32      |11.29.62.238    |simhosta1                  |SUT                 |ai_37   |aidrs_2    |39FB3FAD-800D-53B6-832A-CF20DE2C7004

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |44.55M                        
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |11741561                      
|Total Connections Received                           |6492961                       
|Total Commands Processed                             |93077652                      
|Number of Keys                                       |737                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.0.6                         
|Storage Size                                         |13967917056.0                 
|Data Size                                            |12135843052.0                 
|Index Size                                           |503093808                     
|Average Object Size                                  |6343.86705517                 
|Collections                                          |62                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |5                             
|SVMs                                                 |0                             
|AIs                                                  |5                             
|AIDRSs                                               |1                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |5                             
|VMs ARRIVED                                          |122                           
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |117                           
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |4                             
|SVM RESERVATIONS                                     |0                             
|SVMs ARRIVED                                         |0                             
|SVMs ARRIVING                                        |0                             
|SVMs DEPARTED                                        |0                             
|SVMs DEPARTING                                       |0                             
|SVMs FAILED                                          |0                             
|AI RESERVATIONS                                      |5                             
|AIs ARRIVED                                          |35                            
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |30                            
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |2                             
|AIDRS RESERVATIONS                                   |1                             
|AIDRSs ARRIVED                                       |2                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |1                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |401                           

 ---------------- VMC-WIDE COUNTERS ----------------
|35310FEB-1E59-53F1-A37C-0E4387DFF893 (simzone_a) VM RESERVATIONS|3                             
|DAEF899C-8C95-5B76-BE3F-58349C261E66 (simzone_d) VM RESERVATIONS|0                             
|8E18B916-CC95-566E-887E-3032BA7F835A (simzone_c) VM RESERVATIONS|0                             
|3076C295-3106-5ADC-8C71-D21083902D7A (simzone_b) VM RESERVATIONS|2                             

 ---------------- AIDRS-WIDE COUNTERS ----------------
|86CFEE5E-2911-588B-BD22-2741F64C5DCA (aidrs_2) AI RESERVATIONS|5                             

